src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2004,11 +1907,112 @@ class CgroupUtils(object):
             return os.path.join(subdir, jobid, cgfile)
         return os.path.join(subdir, jobid, (prefix + cgfile))
 
+    # Read the config file in json format
+    @staticmethod
```

Comment:
Rather than use the staticmethod decorator, should we just move the function outside of the class?

Reply:
Because cgroup configurations are stored in `CgroupUtils` class, I think it makes more sense that `parse_config_file` stay in the same class. Convering `parse_config_file` a static function of the class made it possible to call the function without creating an instance.

Reply:
Makes sense.